README.md

```diff
@@ -79,50 +79,61 @@ For help upgrading between major versions, check out the [upgrading documentatio
 
 ## Available helpers
 
-* [Action](#action-helpers)
-  + [`pipe`](#pipe)
-  + [`compute`](#compute)
-  + [`toggle`](#toggle)
-  + [`noop`](#noop)
-  + [`optional`](#optional)
-  + [`queue`](#queue)
-* [Array](#array-helpers)
-  + [`map`](#map)
-  + [`map-by`](#map-by)
-  + [`sort-by`](#sort-by)
-  + [`filter`](#filter)
-  + [`filter-by`](#filter-by)
-  + [`reject-by`](#reject-by)
-  + [`find-by`](#find-by)
-  + [`intersect`](#intersect)
-  + [`invoke`](#invoke)
-  + [`union`](#union)
-  + [`take`](#take)
-  + [`drop`](#drop)
-  + [`reduce`](#reduce)
-  + [`repeat`](#repeat)
-  + [`reverse`](#reverse)
-  + [`range`](#range)
-  + [`join`](#join)
-  + [`compact`](#compact)
-  + [`contains`](#contains)
-  + [`append`](#append)
-  + [`chunk`](#chunk)
-  + [`without`](#without)
-  + [`shuffle`](#shuffle)
-  + [`flatten`](#flatten)
-  + [`object-at`](#object-at)
-  + [`slice`](#slice)
-  + [`next`](#next)
-  + [`has-next`](#has-next)
-  + [`previous`](#previous)
-  + [`has-previous`](#has-previous)
-* [Object](#object-helpers)
-  + [`group-by`](#group-by)
-* [Math](#math-helpers)
-  + [`inc`](#inc)
-  + [`dec`](#dec)
-* [String](#string-helpers)
+- [ember-composable-helpers](#ember-composable-helpers)
+  - [Configuration](#configuration)
```

Comment:
Looks like this is linking to a place above.  Possible we can add this table of contents close to the start of the readme?

Reply:
@snewcomer I have updated readme with table of contents.